device/arista/x86_64-arista_7800r3a_36d2_lc/Arista-7800R3A-36D2-D36/0/port_config.ini

```diff
@@ -0,0 +1,21 @@
+# name         lanes                             alias        index  role       speed       asic_port_name     coreId      corePortId      numVoq
+Ethernet0      72,73,74,75,76,77,78,79           Ethernet1/1  1      Ext        400000      Eth0-ASIC0         1           1               8
+Ethernet8      80,81,82,83,84,85,86,87           Ethernet2/1  2      Ext        400000      Eth8-ASIC0         1           3               8
+Ethernet16     88,89,90,91,92,93,94,95           Ethernet3/1  3      Ext        400000      Eth16-ASIC0        1           5               8
+Ethernet24     96,97,98,99,100,101,102,103       Ethernet4/1  4      Ext        400000      Eth24-ASIC0        1           7               8
+Ethernet32     104,105,106,107,108,109,110,111   Ethernet5/1  5      Ext        400000      Eth32-ASIC0        1           9               8
+Ethernet40     112,113,114,115,116,117,118,119   Ethernet6/1  6      Ext        400000      Eth40-ASIC0        1           11              8
+Ethernet48     120,121,122,123,124,125,126,127   Ethernet7/1  7      Ext        400000      Eth48-ASIC0        1           13              8
+Ethernet56     128,129,130,131,132,133,134,135   Ethernet8/1  8      Ext        400000      Eth56-ASIC0        1           15              8
+Ethernet64     136,137,138,139,140,141,142,143   Ethernet9/1  9      Ext        400000      Eth64-ASIC0        1           17              8
+Ethernet72     64,65,66,67,68,69,70,71           Ethernet10/1 10     Ext        400000      Eth72-ASIC0        0           19              8
+Ethernet80     56,57,58,59,60,61,62,63           Ethernet11/1 11     Ext        400000      Eth80-ASIC0        0           21              8
+Ethernet88     48,49,50,51,52,53,54,55           Ethernet12/1 12     Ext        400000      Eth88-ASIC0        0           23              8
+Ethernet96     40,41,42,43,44,45,46,47           Ethernet13/1 13     Ext        400000      Eth96-ASIC0        0           25              8
+Ethernet104    32,33,34,35,36,37,38,39           Ethernet14/1 14     Ext        400000      Eth104-ASIC0       0           27              8
+Ethernet112    24,25,26,27,28,29,30,31           Ethernet15/1 15     Ext        400000      Eth112-ASIC0       0           29              8
+Ethernet120    16,17,18,19,20,21,22,23           Ethernet16/1 16     Ext        400000      Eth120-ASIC0       0           31              8
+Ethernet128    8,9,10,11,12,13,14,15             Ethernet17/1 17     Ext        400000      Eth128-ASIC0       0           33              8
+Ethernet136    0,1,2,3,4,5,6,7                   Ethernet18/1 18     Ext        400000      Eth136-ASIC0       0           35              8
+Ethernet-Rec0  221                               Recirc0/0    19     Rec        400000      Rcy0-ASIC0         0           221             8
```

Comment:
The index for the `Rec0` and `IB0` overlap with the Ethernet144 and Ethernet152.